application/classes/Ushahidi/Repository/Post.php

```diff
@@ -399,6 +399,23 @@ protected function setSearchConditions(SearchData $search)
 				;
 		}
 
+		if ($sources = $search->source)
```

Comment:
Is this assignment necessary to preserve $search->source? It's a little out of step with the other checks, for consistency would it be possible to use just $search->source?